### PR TITLE
Old edits and XML

### PR DESCRIPTION
**Pull request by owlsandcoffee**
In the future, each branch will focus on one priority. This one includes two:

edits, after our prior meeting. I updated http.py to fix formatting, typos, and include your advice.

Updated naxos.py to make more efficient.

Updated gitignore to prevent sharing of our data.

Added a few MARCXML files to directory \data to use with...

Created xml_cid.py:

This so far, is capable of parsing the xml files. I'm still working to just figure out what lxml can do with MARCXML. Once I do that I will reduce the iterations by using pandas to create a dataframe with the data we want.

### CODE REVIEW

File: src/http.py [hunk 1]
```diff
@@ -1,34 +1,58 @@
 ###Finding records in Sierra that do not have working links###
 
+#10/11: formatted imports to match discussion
 import csv
+#10/11: had to pip install requests in this venv. Resolved
 import requests
 from pymarc import Field,Subfield,MARCReader
 
 class Rowish:
     def __init__(self,control,url,bib_code,suppressed,live):
```
Comment (klinga) [inline src/http.py:10]: I don't think trailing commas are needed in your class properties. I haven't seen them being used. See example here that does not include any trailing commas:
https://docs.python.org/3/tutorial/classes.html#class-objects

Comment (klinga) [reply src/http.py:10]: Let's remove them and commit to this PR

File: src/http.py [hunk 2]
```diff
@@ -1,34 +1,58 @@
...
         classed.append(new_row)
+    #Rowish_named = namedtuple("Rowish_named","control,url,bib_code,suppressed,live")
+    #for row in map(Rowish_named._make,csv.reader(open("file","rb"))):
+    #   classed.append(row)
     return classed
 
 def mrc_classer(file):
     classed = []
     for record in file:
         control = record.get_fields["001"]
```
Comment (klinga) [inline src/http.py:37]: `control = record.get_fields["001"]` should be
`control = record.get_fields("001")`
You are using the .`get_fields()` method (parenthesis).
Square brackets are used when you access a dictionary by a key:
`record["001"]`

**Review by klinga — APPROVED** [review]

File: src/http.py (hunk 2 above)
Comment (owlsandcoffee) [reply src/http.py:37]: Understood! Thank you.

**Pull request merged**